### PR TITLE
update PD's `location-lable`

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Updated descriptions of PD's `location-lable`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7432
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: schedule-replicas-by-topology-labels.md [hunk 1]
```diff
@@ -42,7 +42,15 @@ labels = "zone=<zone>,rack=<rack>,host=<host>"
 
 根据前面的描述，标签可以是用来描述 TiKV 属性的任意键值对，但 PD 无从得知哪些标签是用来标识地理位置的，而且也无从得知这些标签的层次关系。因此，PD 也需要一些配置来使得 PD 理解 TiKV 节点拓扑。
 
-PD 上的配置叫做 `location-labels`，在集群初始化之前，可以通过 PD 的配置文件进行配置。
+PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV 的 `labels` 的 key 是对应的，而且其中每个 key 的顺序代表了不同标签的层次关系。
```
Comment (JmPotato) [inline schedule-replicas-by-topology-labels.md:45]: 感觉最好描述一下从左往右，隔离级别依次递减？

**Review by JmPotato — APPROVED** [review]

File: schedule-replicas-by-topology-labels.md [hunk 2]
```diff
@@ -42,32 +42,36 @@ labels = "zone=<zone>,rack=<rack>,host=<host>"
 
 根据前面的描述，标签可以是用来描述 TiKV 属性的任意键值对，但 PD 无从得知哪些标签是用来标识地理位置的，而且也无从得知这些标签的层次关系。因此，PD 也需要一些配置来使得 PD 理解 TiKV 节点拓扑。
 
-PD 上的配置叫做 `location-labels`，在集群初始化之前，可以通过 PD 的配置文件进行配置。
+PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV 的 `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（从左到右依次递减）。
```
Comment (TomShawn) [inline schedule-replicas-by-topology-labels.md:45]:
```suggestion
PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（从左到右依次递减）。
```

**Review by leowangpingcap — APPROVED** [review]

File: schedule-replicas-by-topology-labels.md [hunk 3]
```diff
@@ -42,32 +42,36 @@ labels = "zone=<zone>,rack=<rack>,host=<host>"
 
 根据前面的描述，标签可以是用来描述 TiKV 属性的任意键值对，但 PD 无从得知哪些标签是用来标识地理位置的，而且也无从得知这些标签的层次关系。因此，PD 也需要一些配置来使得 PD 理解 TiKV 节点拓扑。
 
-PD 上的配置叫做 `location-labels`，在集群初始化之前，可以通过 PD 的配置文件进行配置。
+PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（从左到右依次递减）。
```
Comment (en-jin19) [inline schedule-replicas-by-topology-labels.md:45]:
```suggestion
PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（隔离级别从左到右依次递减）。
```

File: schedule-replicas-by-topology-labels.md [hunk 4]
```diff
@@ -42,32 +42,36 @@ labels = "zone=<zone>,rack=<rack>,host=<host>"
 
 根据前面的描述，标签可以是用来描述 TiKV 属性的任意键值对，但 PD 无从得知哪些标签是用来标识地理位置的，而且也无从得知这些标签的层次关系。因此，PD 也需要一些配置来使得 PD 理解 TiKV 节点拓扑。
 
-PD 上的配置叫做 `location-labels`，在集群初始化之前，可以通过 PD 的配置文件进行配置。
+PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（隔离级别从左到右依次递减）。
```
Comment (en-jin19) [inline schedule-replicas-by-topology-labels.md:45]:
```suggestion
PD 上的配置叫做 `location-labels`，是一个字符串数组。该配置的每一项与 TiKV `labels` 的 key 是对应的，而且其中每个 key 的顺序代表不同标签的级别关系（从左到右，隔离级别依次递减）。
```

**Pull request merged**